README.md

```diff
@@ -1,7 +1,44 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+## Requirements
+
+- Node 18
+- yarn 3.2.4
```

Comment:
Maybe just "yarn v3"? We are explicitly setting the full version in the steps below.